cumulus/parachains/runtimes/assets/asset-hub-westend/src/lib.rs

```diff
@@ -1893,34 +1893,34 @@ impl_runtime_apis! {
 						xcm_config::XcmConfig,
 						ExistentialDepositAsset,
 						PriceForSiblingParachainDelivery,
-						RandomParaId,
+						SiblingAssetHubId,
 						ParachainSystem,
 					>
 				);
 
 				fn reachable_dest() -> Option<Location> {
-					Some(Parent.into())
+					Some(AssetHubWestend::get())
```

Comment:
please, everywhere where you changed **Location** (location and not assetId) `Parent` to `AssetHubWestend`, you need to change `Parent` tot `RandomParaLocation`

```suggestion
					Some(RandomParaLocation::get())
```